tests/validation/tests/v3_api/test_cluster_templates.py

```diff
@@ -4,6 +4,10 @@
 from .test_rbac import create_user
 from rancher import ApiError
 import requests
+from .test_monitoring import validate_cluster_graph
+from .test_monitoring import cluster_query_template
+import copy
```

Comment:
Please group the imports as suggested by Python style:

- Standard library imports.
- Related third party imports.
- Local application/library specific imports.

And each import group sorted alphabetically.

In this case:

```python
import copy
import os
import pytest
import requests
from rancher import ApiError
from .common import *  # NOQA
from .test_monitoring import cluster_query_template
from .test_monitoring import validate_cluster_graph
from .test_rbac import create_user
```

Reply:
Done